### PR TITLE
Panel background brush

### PR DESCRIPTION
**Pull request by amerkoleci**
Add Background to Panel #142

### CODE REVIEW

File: src/Perspex.Controls/Canvas.cs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+		/// <returns>The space taken.</returns>
+		protected override Size ArrangeOverride(Size finalSize)
+		{
+			foreach (Control child in Children)
+			{
+				double x = 0.0;
+				double y = 0.0;
+				double elementLeft = GetLeft(child);
+
+				if (double.IsNaN(elementLeft) == false)
```
Comment (grokys) [inline src/Perspex.Controls/Canvas.cs]: The usual way to express this would be `if (!double.IsNan(elementLeft))`

Comment (amerkoleci) [inline src/Perspex.Controls/Canvas.cs]: I usually prefer the == false condition, but is ok anyway :)

File: src/Perspex.Controls/Canvas.cs [hunk 2]
```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) The Perspex Project. All rights reserved.
```
Comment (grokys) [inline src/Perspex.Controls/Canvas.cs]: Wait... What is Canvas.cs doing in a commit called "Panel Background Brush"? Wasn't Canvas.cs already submitted?

File: src/Perspex.Controls/Panel.cs [hunk 3]
```diff
@@ -6,145 +6,177 @@
 using System.Collections.Specialized;
 using System.Linq;
 using Perspex.Collections;
+using Perspex.Media;
 
 namespace Perspex.Controls
 {
-    /// <summary>
```
Comment (grokys) [inline src/Perspex.Controls/Panel.cs]: Can you fix your editor settings so that your changes don't replace the entire file contents?

**Comment by kekekeks** [comment]
Merged

**Pull request closed without merging**